src/main/webapp/index.html

```diff
@@ -77,6 +81,7 @@ <h1 class="title">🌏project title</h1>
         <p class="close-modal-button" id="close-create-event-modal-button" onclick="closeCreateEventModal()">x</p>
         <form id="create-event-form" action="/events" method="POST">
             <input type="text" id="user-timezone" name="timezone" style="display:none;" required></input>
+            <input type="text" id="id-token" name="id_token" style="display:none;" required></input>
```

Comment:
nit: Hidden required nodes can lead to an unpleasant user experience if some JS fails - the user hits submit and the browser highlights a hidden field, which to the end user looks like the browser does nothing and refuses to do anything.

A better approach would be to drop the required attribute, and return an error response in your servlet. In the future, you may chose to surface the error response to the user if you want.